cypress/integration/tests/migration/controls/tags/tagcategory/create.test.ts

```diff
@@ -67,7 +67,7 @@ describe("Tag type validations", { tags: "@tier2" }, () => {
 
     it("Tag type field validations", function () {
```

Comment:
Tag type needs to be replaced on lines 68, 69 . I also found 2 other instances of 'Tag type ' in this file .

Reply:
1) controls/tags/tagcategory/filter.test.ts  - '5 instances of Tag type'

Reply:
Thanks a lot @nachandr for spotting those. It helps a lot :)
1. controls/tags/tagcategory/filter.test.ts-->Tag type filter validations TC we are removing So didn't change there https://github.com/konveyor/tackle-ui-tests/pull/452 

Rest instances been changed.
Please review let me know if any more

Reply:
@kpunwatk Please initate a jenkins run with this branch full run

Reply:
Done https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/mta-ui-tests-runner/122/console